### DISCUSSION TITLE
Workflow of a feature

### DISCUSSION THREAD

**Discussion by chenrujun** [Microsoft/spring-cloud-azure#980] [Ideas]
Original text has been deleted. Because this will only be use inside **Spring Cloud Azure** team, it's not ready to public to all open-source developers.

For **Spring Cloud Azure** developers, if you want to check the feature workflow, please refer to ~our team's **one note**~ https://github.com/chenrujun/spring-cloud-azure-feature-workflow/issues/1.

**Comment by moarychan** [comment]
Good points! But we are already an agile team now and we should try to improve this with a more agile process.

**Reply by chenrujun** [reply]
> Good points! But we are already an agile team now and we should try to improve this with a more agile process.

@moarychan Thanks for your feedback, I checked the [agile workflow](https://www.atlassian.com/agile/project-management/workflow). IMU, the agile workflow mainly used to monitor the status of an issue (or a task). Current workflow is targeting to a feature. That's why the title is **Workflow of a feature**. A feature workflow needs steps like `Feature Spec review`.
